Automattic/wp-calypso
# Happy Blocks: Fix footer image position

Related to https://github.com/Automattic/wpsupport3/issues/452

## Proposed Changes

Fixes Happy blocks footer image and missing paddings.

Padding issue:
<img width="813" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/252efa3c-ad52-4ab7-89f2-78f02048bfe4">


Misaligned image issue:
<img width="1540" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/25de30fb-d714-4a63-8fa6-e1d09148cbc1">


## Testing Instructions

- Pull and run this branch
- Checkout branch and `yarn dev --sync` from `apps/happy-blocks`
- Check [support](https://wordpress.com/support/) and [learn](https://learncft.wordpress.com/) if:
	-  The paddings around the tablet viewport is 24px like the other elements
	- The image on the desktop is aligned to the right on the footer.
